src/tests/compiletest.md

```diff
@@ -451,8 +451,8 @@ fn main() {
 
 ## Revisions
 
-Certain classes of tests support "revisions" (as of <!-- date: 2021-02 --> February 2021,
-this includes compile-fail, run-fail, and incremental, though
+Certain classes of tests support "revisions" (as of <!-- date: 2022-02 --> February 2022,
+this includes assembly, codegen, incremental, and (rustdoc) UI tests, though
```

Comment:
What about putting UI tests first and separating rustc UI from rustdoc UI tests in the list? I think it'd be clearer.

Reply:
👍, updated as suggested.